### DISCUSSION TITLE
Add link shortening

### DISCUSSION THREAD

**Discussion by supervova** [alangrainger/share-note#141] [Vote for upcoming features]
https://share.note.sx/ch6qb2sd#BxcHSBpvIJK3x0vKPpx3kS0+nVL5KTQ+2Vw1q3SF8rs  → https://share.note.sx/ch6qb

**Comment by alangrainger** [comment]
Just don't encrypt a note and then you will only have a short link, like this: 

https://share.note.sx/4h0yoc12

If I did the way you're suggesting, then I would have the decryption key stored on the server, which would negate the encryption.
